Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 8428
"GET /static/simulator/css/style.css HTTP/1.1" 200 8591
"GET /static/simulator/js/main.js HTTP/1.1" 200 5434
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 4454
"GET /admin/login/ HTTP/1.1" 200 4218
"GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
"GET /static/admin/js/theme.js HTTP/1.1" 200 1653
"GET /static/admin/css/base.css HTTP/1.1" 200 22154
"GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
"GET /static/admin/css/login.css HTTP/1.1" 200 951
"GET /static/admin/css/responsive.css HTTP/1.1" 200 16632
"GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
"GET / HTTP/1.1" 200 8428
"GET / HTTP/1.1" 200 8428
"GET / HTTP/1.1" 200 8428
"GET / HTTP/1.1" 200 8428
"GET / HTTP/1.1" 200 8409
"GET /static/simulator/js/main.js HTTP/1.1" 304 0
"GET /static/simulator/css/style.css HTTP/1.1" 304 0
"GET /admin/login/ HTTP/1.1" 200 4218
"POST /admin/login/ HTTP/1.1" 302 0
Internal Server Error: /admin/
Traceback (most recent call last):
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 265, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 562, in index
    app_list = self.get_app_list(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 546, in get_app_list
    app_dict = self._build_app_dict(request, app_label)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 530, in _build_app_dict
    "app_url": reverse(
               ^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\urls\base.py", line 98, in reverse
    resolved_url = resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'app_list' with keyword arguments '{'app_label': 'simulator'}' not found. 1 pattern(s) tried: ['admin/(?P<app_label>auth)/$']
"GET /admin/ HTTP/1.1" 500 119176
"GET /admin/login/ HTTP/1.1" 302 0
"GET / HTTP/1.1" 200 9028
"GET /dashboard/ HTTP/1.1" 302 0
"GET /profile/create/ HTTP/1.1" 200 9538
"POST /profile/create/ HTTP/1.1" 302 0
Internal Server Error: /dashboard/
Traceback (most recent call last):
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\auth\decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\STUDIA\ZZ_Inne\HackYeah\repo\RetirementSimulator\simulator\views.py", line 124, in dashboard
    return render(request, 'simulator/dashboard.html', context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\loader_tags.py", line 299, in do_extends
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\template\base.py", line 567, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 411: 'static', expected 'endblock'. Did you forget to register or load this tag?
"GET /dashboard/ HTTP/1.1" 500 194901
"POST /profile/create/ HTTP/1.1" 302 0
"GET /dashboard/ HTTP/1.1" 200 19072
"GET /static/simulator/css/style.css HTTP/1.1" 304 0
"GET /static/simulator/js/main.js HTTP/1.1" 304 0
"GET /static/simulator/js/timeline.js HTTP/1.1" 200 17034
Forbidden (CSRF token missing.): /ajax/update-retirement-year/
"POST /ajax/update-retirement-year/ HTTP/1.1" 403 2524
Forbidden (CSRF token missing.): /ajax/update-retirement-year/
"POST /ajax/update-retirement-year/ HTTP/1.1" 403 2524
Forbidden (CSRF token missing.): /ajax/update-retirement-year/
"POST /ajax/update-retirement-year/ HTTP/1.1" 403 2524
Forbidden (CSRF token missing.): /ajax/update-retirement-year/
"POST /ajax/update-retirement-year/ HTTP/1.1" 403 2524
"POST /ajax/work-period/ HTTP/1.1" 200 139
"GET /ajax/calculate-pension/ HTTP/1.1" 200 200
"GET /dashboard/ HTTP/1.1" 200 18903
D:\STUDIA\ZZ_Inne\HackYeah\repo\RetirementSimulator\simulator\views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /dashboard/
Traceback (most recent call last):
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\STUDIA\ZZ_Inne\HackYeah\repo\RetirementSimulator\simulator\views.py", line 48, in dashboard
    current_age = profile_data.get('age', 30) if hasattr(profile_data, 'get') else profile_data['age']
                                                                                   ~~~~~~~~~~~~^^^^^^^
TypeError: 'UserProfile' object is not subscriptable
"GET /dashboard/ HTTP/1.1" 500 68563
"GET /profile/create/ HTTP/1.1" 302 0
Internal Server Error: /admin/logout/
Traceback (most recent call last):
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 265, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\utils\decorators.py", line 192, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\utils\decorators.py", line 190, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 246, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 400, in logout
    **self.each_context(request),
      ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 343, in each_context
    "available_apps": self.get_app_list(request),
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 546, in get_app_list
    app_dict = self._build_app_dict(request, app_label)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\contrib\admin\sites.py", line 530, in _build_app_dict
    "app_url": reverse(
               ^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\urls\base.py", line 98, in reverse
    resolved_url = resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\flang\anaconda3\envs\retsimulator\Lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'app_list' with keyword arguments '{'app_label': 'simulator'}' not found. 1 pattern(s) tried: ['admin/(?P<app_label>auth)/$']
"GET /admin/logout/ HTTP/1.1" 500 123827
D:\STUDIA\ZZ_Inne\HackYeah\repo\RetirementSimulator\RetirementSimulator\urls.py changed, reloading.
Watching for file changes with StatReloader
D:\STUDIA\ZZ_Inne\HackYeah\repo\RetirementSimulator\RetirementSimulator\urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 14147
"GET /static/simulator/css/style.css HTTP/1.1" 200 8591
"GET /static/simulator/js/main.js HTTP/1.1" 200 5434
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 4625
"GET /admin/login/ HTTP/1.1" 200 4218
"GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2808
"GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810
"GET /static/admin/css/login.css HTTP/1.1" 200 951
"GET /static/admin/js/theme.js HTTP/1.1" 200 1653
"GET /static/admin/css/base.css HTTP/1.1" 200 22154
"GET /static/admin/css/responsive.css HTTP/1.1" 200 16632
"GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 14137
"GET /static/simulator/css/style.css HTTP/1.1" 304 0
"GET /static/simulator/js/main.js HTTP/1.1" 304 0
